src/Framework/AbstractConfig.php

```diff
@@ -20,7 +20,14 @@ protected function get(string $key, $default = Config::DEFAULT_CONFIG_VALUE)
         return Config::getInstance()->get($key, $default);
     }
 
-    protected function getAppRootDir(): string
+	/**
+	 *
+	 * Allow easy access the root dir of the project from the Factory
+	 *
+	 * @return string
```

Comment:
If the types are already at code level (for argument types or returning types) we don't need/want to duplicate and define the return type again as PHPDoc. We use PHPDoc types only for array/list, generics, or when we want to provide a description to it.